dmcontent/questions.py

```diff
@@ -626,6 +662,20 @@ def answer_required(self):
             return self.is_empty
 
 
+class DateSummary(QuestionSummary):
+
+    def __init__(self, question, service_data):
+        super(DateSummary, self).__init__(question, service_data)
+        self._value = self._service_data.get(self.id, '')
+
+    @property
+    def value(self):
+        try:
+            return datetime.strptime(self._value, '%Y-%m-%d').strftime('%A %-d %B %Y')
```

Comment:
There's a reason we we've been avoiding that. Has to do with having to add it as a dependency link in the setup.py as `digitalmarketplace-content-loader` is a module. I've added it, we can chat on Friday.